files/en-us/learn/forms/form_validation/index.html

```diff
@@ -411,10 +411,11 @@ <h3 id="The_Constraint_Validation_API">The Constraint Validation API</h3>
  <li><code>willValidate</code>: Returns <code>true</code> if the element will be validated when the form is submitted; <code>false</code> otherwise.</li>
 </ul>
 
-<p id="Constraint_validation_API_methods">The Constraint Validation API also makes the following methods available on the above elements.</p>
+<p id="Constraint_validation_API_methods">The Constraint Validation API also makes the following methods available on the above elements and the <code><a href="/en-US/docs/Web/HTML/Element/form">form</a></code> element.</p>
 
 <ul>
  <li><code>checkValidity()</code>: Returns <code>true</code> if the element's value has no validity problems; <code>false</code> otherwise. If the element is invalid, this method also fires an <a href="/en-US/docs/Web/API/HTMLInputElement/invalid_event"><code>invalid</code> event</a> on the element.</li>
+ <li><code>reportValidity()</code>: Report the invalid field(s) using the browser native messages. Useful in combination with <code>preventDefault()</code> in an <code>onSubmit</code> event handler</li>
```

Comment:
The key point of reportValidity is not the native messages, it is the events.
```suggestion
 <li><code>reportValidity()</code>: Reports invalid field(s) using events. Useful in combination with <code>preventDefault()</code> in an <code>onSubmit</code> event handler</li>
```

Reply:
I'm going ahead, commiting and merging this, as, with tonight migration to Markdown, this would have to be redone.